### DISCUSSION TITLE
GSOC '23 Applicant

### DISCUSSION THREAD

**Discussion by just-injoey** [openvinotoolkit/openvino#15954] [Google Summer of Code]
Dear OpenVINO developers,
I am Justin T Joseph,  CSE undergrad from SRM University.
I'm interested in contributing to OpenVINO and want to get connected with the mentors for GSOC '23. I'd like to know and work on any open issues to get started and know more about the community.
So pls suggest me what further steps should i take.
Hope that you will take my request into consideration.

Regards,
Justin

**Comment by adrianboguszewski** [comment]
Hi @just-injoey, please run [OpenVINO notebooks](https://github.com/openvinotoolkit/openvino_notebooks) first and learn some key aspects from it and then start working on the [prerequisite task](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#prerequisite-task) :)
